lib/config/configItems.joi.js

```diff
@@ -150,6 +151,10 @@ const qpKafkaJoi = kafkaJoi.append({
     replication: extensionKafkaJoi,
 });
 
+const supportedLifecycleRulesJoi =  joi.array().items(
```

Comment:
```suggestion
const supportedLifecycleRulesJoi = joi.array().items(
```

Reply:
(maybe not very useful to store this one here if we only use it once, but the array can stay here)